eclair-core/src/test/scala/fr/acinq/eclair/wire/protocol/PaymentOnionSpec.scala

```diff
@@ -82,17 +82,18 @@ class PaymentOnionSpec extends AnyFunSuite {
   test("encode/decode variable-length (tlv) relay per-hop payload") {
     val testCases = Map(
       TlvStream[OnionPaymentPayloadTlv](AmountToForward(561 msat), OutgoingCltv(CltvExpiry(42)), OutgoingChannelId(ShortChannelId(1105))) -> hex"11 02020231 04012a 06080000000000000451",
-      TlvStream[OnionPaymentPayloadTlv](AmountToForward(561 msat), OutgoingCltv(CltvExpiry(42)), OutgoingChannelId(ShortChannelId(1105)), EncryptedRecipientData(hex"0123456789abcdef"), BlindingPoint(PublicKey(hex"036d6caac248af96f6afa7f904f550253a0f3ef3f5aa2fe6838a95b216691468e2"))) -> hex"3e 02020231 04012a 06080000000000000451 0a080123456789abcdef 0c21036d6caac248af96f6afa7f904f550253a0f3ef3f5aa2fe6838a95b216691468e2",
```

Comment:
We shouldn't just delete this, we should replace it with a new test vector that contains valid `encrypted_data`, shouldn't we?

Reply:
I can't because this test checks the values of `amountToForward`, `outgoingCltv` and `outgoingChannelId` which are not available in the case of a blinded route. I've added back this test case in its own test for blinded routes.